### PR TITLE
Add command to register private controller

### PR DESCRIPTION
**Pull request by hemanthnakkina**
Add new commands to register and unregister private juju controller.
    
```
sunbeam juju register-controller NAME TOKEN [--force]
sunbeam juju unregister-controller NAME
```

### CODE REVIEW

File: sunbeam-python/sunbeam/commands/juju_utils.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+LOG = logging.getLogger(__name__)
+console = Console()
+
+
+@click.command()
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    help="Replace if controller exists with same name",
```
Comment (javacruft) [inline sunbeam-python/sunbeam/commands/juju_utils.py:35]: "Force replacement if controller already exists with the same name"

**Comment by javacruft** [comment]
I think the UX is fine - this mirrors register/unregister in Juju itself and as an advanced usage case, the operator will most likely be familiar with Juju anyway.

File: sunbeam-python/sunbeam/commands/juju_utils.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+)
+@click.argument("name", type=str)
+@click.argument("token", type=str)
+@click.pass_context
+def register_controller(ctx: click.Context, name: str, token: str, force: bool) -> None:
+    """Register private controller."""
```
Comment (javacruft) [inline sunbeam-python/sunbeam/commands/juju_utils.py:41]: Register existing Juju controller

File: sunbeam-python/sunbeam/commands/juju_utils.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    run_plan(plan, console)
+    console.print(f"Controller {name} registered")
+
+
+@click.command()
+@click.argument("name", type=str)
+@click.pass_context
+def unregister_controller(ctx: click.Context, name: str) -> None:
+    """Unregister private controller."""
```
Comment (javacruft) [inline sunbeam-python/sunbeam/commands/juju_utils.py:58]: Unregister external Juju controller

**Comment by javacruft** [comment]
I tweak the text around the click command setup but apart from that LGTM

File: sunbeam-python/sunbeam/commands/juju_utils.py (hunk 1 above)
Comment (hemanthnakkina) [reply sunbeam-python/sunbeam/commands/juju_utils.py]: Fixed

File: sunbeam-python/sunbeam/commands/juju_utils.py (hunk 2 above)
Comment (hemanthnakkina) [reply sunbeam-python/sunbeam/commands/juju_utils.py]: Fixed

File: sunbeam-python/sunbeam/commands/juju_utils.py (hunk 3 above)
Comment (hemanthnakkina) [reply sunbeam-python/sunbeam/commands/juju_utils.py]: Fixed

**Review by javacruft — APPROVED** [review]

**Pull request merged**